mobile/README.md

```diff
@@ -35,6 +35,25 @@
 - Change `storeFile` variable in `key.properties` file to point to where you put `airqo-dev-keystore.jks`.
 For example if `airqo-dev-keystore.jks` is located under `/Users/example/` then `storeFile=/Users/example/airqo-dev-keystore.jks`
 
+- **For Contributors:**
+  - It's important to first read our guidelines [here](/../CONTRIBUTING.md).
+  - Thereafter, proceed to create your own necessary configuration files. These include;
+    - Add an `.env file`, which contain credentials in key-value format for services used by the app. For development, save the file as `.env.dev` while for production, save it as `.env.prod`. 
+      - Here's what your .env file would look like;
+        ```bash
+        AIRQO_API_URL= https://platform.airqo.net/api/v1/
+        AIRQO_API_URL_V2= https://platform.airqo.net/api/v2/
+        AIRQO_API_TOKEN = Get this token by following the respective guide below.
+        GOOGLE_ANDROID_API_KEY_= Get this by following the respective guide below.
+        GOOGLE_IOS_API_KEY_v1 = Get this key by following the respective guide below.
```

Comment:
I don't think these two should be part of the .env @BenjaminSsempala

Reply:
Thank you. I've removed them @NoahNsimbe